packages/server/lib/automation/commands/key_press.ts

```diff
@@ -60,29 +40,32 @@ export async function cdpKeyPress (
   }
 
   try {
-    await send('Input.dispatchKeyEvent', {
-      type: 'keyDown',
-      key,
-      code: key,
-      keyIdentifier,
-    })
-
-    await send('Input.dispatchKeyEvent', {
-      type: 'keyUp',
-      key,
-      code: key,
-      keyIdentifier,
-    })
+    // Named keys must be dispatched as single characters,
+    // multi-codepoint characters must be dispatched as individual codepoints
+    const isNamedKey = NamedKeys.includes(key)
+
+    const chars = [...key].length === 1 || isNamedKey ? [key] : [...key]
```

Comment:
Can this code be simplified to improve performance?


```suggestion
    const chars = isNamedKey ? [key] : [...key];
```

Reply:
I doubt there would be any measurable performance here. The suggestion also doesn't have the same effect